rpc/client.go

```diff
@@ -21,13 +21,20 @@ import (
 	"net"
 	"net/rpc"
 
+	"github.com/pkg/errors"
+	mux "github.com/xtaci/smux"
+
 	"github.com/CovenantSQL/CovenantSQL/crypto/etls"
+	"github.com/CovenantSQL/CovenantSQL/crypto/hash"
 	"github.com/CovenantSQL/CovenantSQL/crypto/kms"
 	"github.com/CovenantSQL/CovenantSQL/pow/cpuminer"
 	"github.com/CovenantSQL/CovenantSQL/proto"
 	"github.com/CovenantSQL/CovenantSQL/utils"
 	"github.com/CovenantSQL/CovenantSQL/utils/log"
-	mux "github.com/xtaci/smux"
+)
+
+const (
+	ETLSHeaderSize = 2 + hash.HashBSize + 32
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const ETLSHeaderSize should have comment (or a comment on this block) or be unexported